notebooks/Emukit-task-spread-of-disease-with-seir-model.ipynb

```diff
@@ -0,0 +1,908 @@
+    "\n",
+    "### 4. Other \n",
+    "There can be many more questions asked. With a [multi-fidelity model](Emukit-tutorial-multi-fidelity.ipynb) one can relate the high-fidelity Gillespie results to the ODE approximation or we can benchmark different emulators for Task 3.\n",
+    "\n",
+    "If our disease model also included the effect of e.g., vaccines or treatment in a certain area (the SEIR model presented here does not), with [Bayesian optimization](Emukit-tutorial-Bayesian-optimization-introduction.ipynb) we could find the optimal allocation of medical personal in a region.\n",
+    "\n",
+    "The emulator usually also provides gradients of the learned relation (e.g., between infection rate and expected height of peak) which can be used for model-based [sensitivity analysis](Emukit-tutorial-sensitivity-montecarlo.ipynb). \n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Navigation\n",
+    "\n",
+    "1. [Task 1: Height if peak of infection](#1.-Load your objective, collect some data, build a model)\n",
+    "\n",
+    "2. [Task 2: Time-occurance of peak of infection](#2.-Load the elements to solve your problem and run the decision loop if needed)\n",
+    "\n",
+    "3. [Task 3: Infection rate-peak height relation](#3.-Conclusions)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Load Emukit packages\n",
+    "\n",
+    "We use the Emukit  quadrature package for Tasks 1 and 2, and the Emukit experimental design package for Task 3. For common functionality we use Emukit core. For the emulator model we use GPy since GPy model wrappers already exist in Emukit."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# SEIR model and Gillespie simulation\n",
+    "import Emukit_task_seir_model as seir\n",
+    "\n",
+    "# GPy for the model\n",
+    "import GPy\n",
+    "\n",
+    "# Emukit model wrapper\n",
+    "from emukit.model_wrappers.gpy_quadrature_wrappers import RBFGPy, BaseGaussianProcessGPy\n",
+    "from emukit.model_wrappers import GPyModelWrapper\n",
+    "\n",
+    "# Emukit core\n",
+    "from emukit.core.loop.user_function import UserFunctionWrapper\n",
```

Comment:
Outer loop has a small flexibility in here: you can pass just the function itself, and it will wrap it in UserFunction object. May simplify code here a bit.